documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -195,6 +207,12 @@ The initial configuration may be selected according to these high-level guidelin
   compute cores is key for better performance. To allow sufficient work to satisfy all
   X\ :sup:`e`-cores [#fnote2]_ on a GPU, use ``param.data_per_workitem * param.workgroup_size ≈ N / xe_core_count``.
 
+- For large inputs that fully saturate compute cores, maximizing ``param.workgroup_size`` and ``param.data_per_workitem``
+  without spilling out of register memory results in best performance. The Intel® oneAPI DPC++ Compiler reports warnings
+  when register spillage occurs. This may be used alongside guidance provided in the
+  `oneAPI GPU Optimization Guide <https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2025-2/registers-and-performance.html>`_
+  and benchmarking parameter sweeps to determine performant kernel template parameters for your use case.
```

Comment:
What does "benchmarking parameter sweeps" mean? Did you mean "benchmarking parameter sets"?

Reply:
The term is searchable, and the meaning is understandable even without it. Thanks for the clarification, and let's leave it as is. I just have not seen this term before.